app/konnect/dev-portal/access-and-approval/sso.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Configure SSO for Dev Portal
+content_type: how-to
+---
+
+You can configure single sign-on (SSO) for {{site.konnect_short_name}} Dev Portal with OIDC. This allows developers to log in to Dev Portals by using their SSO credentials. 
+
+<!--Do developers get auto approved in Dev Portal then?-->
+
+<!-- would we recommend SSO instead of adding developers and teams from IdPs if we don't want to map team? Or would we maybe recommend both (one to create/map devs and the other to let them log in to Dev Portal)-->
+
+<!-- should we add a note that this SSO is different than the one for Konnect? As in, if they want to use SSO for logging in to Konnect, they still need to configure that separately?-->
```

Comment:
👍 That could be a good not with a link to the configuring in Konnect. Also could be good to note that each Dev Portal has a separate SSO configuration. They can use the same IDP for multiple portals or different IDPs per portal.